static/css/marble.css

```diff
@@ -0,0 +1,399 @@
+/*Fonts*/
+body .body-1{
+    font-family: 'Helvetica Neue', Helvetica, Arial,  sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    letter-spacing: 0.03px;
+}
+
+h1{
+    font-family: Arial, Helvetica, sans-serif;
+    font-weight: 300;
+    font-size: 96px;
+    letter-spacing: -1.44px;
+}
+
+h2{
+    font-family: 'Helvetica Neue', Helvetica, Arial,  sans-serif;
+    font-weight: 300;
+    font-size: 60px;
+    letter-spacing: -0.3px;
+}
+
+h3{
+    font-family: 'Helvetica Neue', Helvetica, Arial,  sans-serif;
+    font-weight: 400;
+    font-size: 48px;
+}
+
+h4{
+    font-family: 'Helvetica Neue', Helvetica, Arial,  sans-serif;
+    font-weight: 400;
+    font-size: 34px;
+    letter-spacing: 0.085px;
+}
+
+h5{
+    font-family: 'Helvetica Neue', Helvetica, Arial,  sans-serif;
+    font-weight: 400;
+    font-size: 24px;
+}
+
+h6{
+    font-family: 'Helvetica Neue', Helvetica, Arial,  sans-serif;
+    font-weight: 500;
+    font-size: 24px;
+    letter-spacing: 0.03px;
+}
+
+.subtitle-1 .leading-text-1{
+    font-family: 'Helvetica Neue', Helvetica, Arial,  sans-serif;
+    font-weight: 500;
+    font-size: 16px;
+    letter-spacing: 0.03px;
+}
+
+.subtitle-2 .leading-text-2{
+    font-family: 'Helvetica Neue', Helvetica, Arial,  sans-serif;
+    font-weight: 400;
+    font-size: 14px;
+    letter-spacing: 0.014px;
+}
+
+.button-text{
+    font-family: 'Helvetica Neue', Helvetica, Arial,  sans-serif;
+    font-weight: 500;
+    font-size: 14px;
+    text-transform: uppercase;
+    letter-spacing: 0.024px;
+}
+
+.caption{
+    font-family: 'Helvetica Neue', Helvetica, Arial,  sans-serif;
+    font-weight: 400;
+    font-size: 12px;
+    letter-spacing: 0.048px;
+}
+
+.overline{
+    font-family: 'Helvetica Neue', Helvetica, Arial,  sans-serif;
+    font-weight: 400;
+    font-size: 10px;
+    text-transform: uppercase;
+    letter-spacing: 0.15px;
+}
+
+/*UI Components*/
+/*Buttons*/
+
+.disabled {
+    opacity: 0.6;
+    cursor: not-allowed;
+    pointer-events: none;
+  }
+
+.button-sm{
+    width: 75px;
+    height: 51px;
+    flex-shrink: 0;
+    border: none;
+}
+
+.button-med{
+    width: 108px;
+    height: 51px;
+    flex-shrink: 0;
+    border: none;
+}
+
+.button-sm:hover, .button-med:hover{
+    background-color: #000000;
+}
+
+.button-med-alt{
+    width: 108px;
+    height: 51px;
+    flex-shrink: 0;
+    border: none;
+}
+
+.button-single-action{
+    width: 48px;
+    height: 48px;
+    flex-shrink: 0;
+    border-radius: 50px;
+    border: none;
+}
+
+/*Colour Palette*/
+/*NOTE: Some colour classes have multiple classes that include those for components above.*/
+
+.palette-background{
+    background-color: #FFFFFF;
+}
+
+.surface{
+    background-color: #FFFFFF;
+}
+
+.surface-alt{
```

Comment:
Is this meant to be empty?

Reply:
I had a conversation with Shruti last night and this morning about the alternative design for the button.  It uses a different colour scheme but the colours are not defined as a primary alternative or surface alternative.  Shruti says the design can only have one primary and one surface colour defined, but an alternative can be defined in the CSS if needed.  I left this blank for the time being.

Reply:
Ok, that makes sense. Any time you do something like this though please leave a comment in the code explaining why.

Reply:
Since last update the surface-alt class is no longer needed.  It is deleted.